src/utils.ts

```diff
@@ -99,3 +99,12 @@ export function intersectMany<T>(arrays: T[][]): T[] {
     return count === arrays.length;
   });
 }
+
+export const includes = <T>(array: T[] | readonly T[], element: T): boolean => {
```

Comment:
If we want to use the `includes` function instead of the built-in `Array.includes` method, I'd recommend turning this into a binary search function. Also, please export a function instead of a constant (for code style consistency)